src/ext/mod.rs

```diff
@@ -10,20 +10,23 @@ pub mod rustyscript;
 trait ExtensionTrait<A> {
     fn init(options: A) -> Extension;
 
-    /// Makes a call to `init_ops_and_esm` equivalent to `init_ops`
-    fn set_esm(mut ext: Extension, is_snapshot: bool) -> Extension {
-        if is_snapshot {
-            ext.js_files = ::std::borrow::Cow::Borrowed(&[]);
-            ext.esm_files = ::std::borrow::Cow::Borrowed(&[]);
-            ext.esm_entry_point = ::std::option::Option::None;
-        }
+    // Clears the js and esm files for warmup to avoid reloading them

```

Comment:
I'm skeptical this is still necessary post-https://github.com/denoland/deno_core/commit/aa6f360059eff374a9105c01c28f2d3846f6ce9c (In fact, I think this entire trait can probably go away in favor of calling Extension::init directly)